airbyte-integrations/connectors/source-amazon-ads/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "5.0.9"
+version = "5.0.10"
```

Comment:
I think we typically count CDK version bumps as minor version bumps for connectors.